src/main/java/com/kodilla/ecommercee/domain/OrderEntity.java

```diff
@@ -22,5 +23,24 @@
     @GeneratedValue
     private long id;
 
+    @GeneratedValue
+    @NotNull
+    private int orderNumber;
+
     private String productGroup;
+    private int price;
+
+    @OneToMany(
+            targetEntity = Product.class,
```

Comment:
Brakuje tutaj parametru mappedBy do pola z klasy Product, które jest po drugiej stronie relacji tj ManyToOne.

Reply:
zgadzam się z @pastorek89

Reply:
Otóż - celowo usunąłem mappedBy, bo z mappedBy nie chce się skompilować. Zgłasza się błąd:
Associations marked as mappedBy must not define database mappings like @JoinTable or @JoinColumn: com.kodilla.ecommercee.domain.OrderEntity.productList

Reply:
zgłasza Ci, że używasz jednocześnie "mappedBy" oraz @JoinColumn... a to jest bez sensu. Więc jedno trzeba usunąć - pytanie czy zespół zdecydował się na użycie mappedBy czy JoinColumn... z tego co pamiętam to w większości wypadków był mappedBy.